### ISSUE TITLE
Item Duplication Bug with Bot Inventory

### ISSUE THREAD

**Issue by shinysocks** [JoeStrout/Farmtronics#12]
- When a player's tool is missing from their inventory and instead in a bot, ex. having two watering cans in a bot, the game thinks the item was lost and gives you another in the lost and found box at Mayor Lewis' house. 
- My guess would be that the game doesn't know to check in the bots before assuming the item was lost in the mines?
- Not a detrimental bug but it does allow for some unfair item duplication if you continue to pick up the new items from the Lost and Found.

**Comment by JoeStrout** [comment]
Yes, I think this is a separate issue.  I don't think #8 still happens anymore.  But this one is going to be thornier to solve — as you say, the game doesn't really know about bot inventories.
